Good-Bids/goodbids
# [N/A] Attempt to fix Bidding Block error.

# Summary

This PR is my attempt to resolve the error on Develop where the Auction ID is not getting read correctly out of the block data attributes. Since this error does not show up locally, this is sort of a fingers-crossed type fix to see if it resolves it on the Development site.

## Issues

* N/A

## Testing Instructions

Upon Deployment:
1. Remove the Bidding Block from the Charity Water: [React Bidding Auction](https://goodbids-develop.go-vip.net/charity-water/wp-admin/post.php?post=295&action=edit)
2. Re-add the Bidding block
3. Update the Auction
4. View the Auction on the front-end to see if the error is still there.
